.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,18 @@
+name: Semgrep
+
+on:
+  pull_request: {}
+  push:
+    branches: ["master"]
```

Comment:
Is it possible to specify `master` and all release branches `[0-9]*` ?

Reply:
Updated. This should be backported to older branches.